### ISSUE TITLE
pyright CI jobs are failing

### ISSUE THREAD

**Issue by danieleades** [python/typeshed#11474]
error:

```
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module './impl/format'
Require stack:
- /home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:[19](https://github.com/python/typeshed/actions/runs/8062628621/job/22022768991?pr=11473#step:10:20))
    at require (node:internal/modules/helpers:130:18)
    at /home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js:4282:25
    at /home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js:4272:17
    at node_modules/jsonc-parser/lib/umd/main.js (/home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js:4278:7)
    at __require (/home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js:12:50)
    at Object.<anonymous> (/home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js:8109:35)
    at Module._compile (node:internal/modules/cjs/loader:1241:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/jakebailey/pyright-action/v2/dist/index.js'
  ]
}

Node.js v[20](https://github.com/python/typeshed/actions/runs/8062628621/job/22022768991?pr=11473#step:10:21).8.1
```

**Comment by AlexWaygood** [comment]
Cc. @jakebailey -- looks like this could be an issue in the 2.1.0 release of pyright-action you cut recently? I'll try pinning to pyright-action 2.0.2 to see if that fixes things

**Comment by AlexWaygood** [comment]
Confirmed that pinning to 2.0.2 fixes the failures: #11475

**Comment by jakebailey** [comment]
I just fixed it in v2.1.1, and added a smoke test which should ensure that this doesn't break again. My fault for not testing the bundle itself (and so I hit https://github.com/microsoft/node-jsonc-parser/issues/57).

**Comment by AlexWaygood** [comment]
Thanks @jakebailey! All green again now
